src/main/java/teammates/ui/controller/AdminSessionsPageAction.java

```diff
@@ -117,25 +114,24 @@ private ActionResult createShowPageResultIfParametersInvalid() {
             SanitizationHelper.sanitizeForHtml(endMin);
             SanitizationHelper.sanitizeForHtml(timeZone);
 
-            zone = Double.parseDouble(timeZone);
+            zone = TimeHelper.convertToZoneId(Double.parseDouble(timeZone));
```

Comment:
I think we should change the front end to send proper time zone id instead of still using double?

Reply:
Also, parsing may fail and throw an exception.